### PR TITLE
 Power BI + ODBC docs

### PR DESCRIPTION
**Pull request by slvrtrn**
preview https://clickhouse-docs-499k7sm27-clickhouse.vercel.app/docs/en/integrations/powerbi or see the screenshot in the first comment.

### CODE REVIEW

**Comment by slvrtrn** [comment]
That's what I've got for ODBC atm.

screencapture-localhost-3000-docs-en-integrations-powerbi-2023-05-08-20_12_56

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 1699e5d2f7229092e11c806d188c170d872c1726 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1131/1699e5d2f7229092e11c806d188c170d872c1726/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

**Comment by DanRoscigno** [comment]
live preview https://clickhouse-docs-499k7sm27-clickhouse.vercel.app/docs/en/integrations/powerbi

File: docs/en/integrations/data-visualization/powerbi-and-clickhouse.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+<br/>
+
+Finally, you should see the databases and tables in the Navigator view. Select the desired table and click "Load" to import the data from ClickHouse.
+
+<img src={require('./images/powerbi_10.png').default} class="image" alt="Navigator view" style={{width: '40%', 'background-color': 'transparent'}}/>
+<br/>
+
+Once the import is complete, your ClickHouse Data should accessible in Power BI as usual.
+
+:::note
```
Comment (mshustov) [inline docs/en/integrations/data-visualization/powerbi-and-clickhouse.md:78]: Can we move it into a separated step to emphasize its importance?

File: docs/en/integrations/data-visualization/powerbi-and-clickhouse.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+sidebar_label: Power BI
+slug: /en/integrations/powerbi
+keywords: [clickhouse, powerbi, connect, integrate, ui]
+description: Microsoft Power BI is an interactive data visualization software product developed by Microsoft with a primary focus on business intelligence.
+---
+import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_http.mdx';
+
+# Power BI
+
+Power BI can load the data from ClickHouse Cloud or on-premise deployment using the [ODBC driver](https://github.com/ClickHouse/clickhouse-odbc) on a Windows machine.
```
Comment (mshustov) [inline docs/en/integrations/data-visualization/powerbi-and-clickhouse.md:11]: Should we call out Windows a prerequisite?

**Review by mshustov — APPROVED** [review]

**Comment by DanRoscigno** [comment]
@slvrtrn is this ready to be merged?

**Comment by DanRoscigno** [comment]
I will check the links and stuff and merge

**Review by DanRoscigno — APPROVED** [review]

**Pull request merged**